Framework/DataObjects/src/PeakColumn.cpp

```diff
@@ -53,6 +53,7 @@ const std::string typeFromName(const std::string &name) {
         TYPE_INDEX.emplace("DSpacing", "double");
         TYPE_INDEX.emplace("Intens", "double");
         TYPE_INDEX.emplace("SigInt", "double");
+        TYPE_INDEX.emplace("Intens/SigInt", "double");
```

Comment:
Ah yes thanks for catching that. I think it doesn't actually matter because it's a one time init so the size is either `0` or the full size but as you say it's better that it's not out of sync.